### PR TITLE
#222 Removed COE Console

### PR DESCRIPTION
**Pull request by PandoraQS**
- The COE Console has been removed
- The link for the COE Console has been changed to a button to start and stop the COE

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@PandoraQS A few suggestions.

1. Please add / update the README to specify correct build instructions
2. Add unit and integration tests for the code you are writing / updating

**Comment by PandoraQS** [comment]
- The README was properly detailed, I removed "gulp init" from "first_steps.rst" as it is outdated and not required to build.
- I wrote tests for the Start COE button

File: src/CoeServerStatusUiController.ts [hunk 1]
```diff
@@ -124,8 +124,29 @@ export class CoeServerStatusUiController {
...
+    public updateButtonState() {
+        var coe = IntoCpsApp.getInstance().getCoeProcess();
+        console.log("is running: ", coe.isRunning());
+        let button = <HTMLButtonElement>document.getElementById("coe-btn-launch-bottom");
+        let icon = <HTMLButtonElement>document.getElementById("coeIconColor");
+    
+        if (coe.isRunning()) {
+            button.innerHTML = 'Stop COE <span id="coeIconColor" style="color:green" class="glyphicon glyphicon-one-fine-dot"></span>';
+            console.log("button: ", button);
+            button.setAttribute('onclick', 'coeViewController.stopCoe()');
```
Comment (prasadtalasila) [inline src/CoeServerStatusUiController.ts:140]: Coe --> CoE

Please use the same convention in other places

**Review by prasadtalasila — COMMENTED** [review]
Please add DEVELOPER.md at [top-level](https://github.com/INTO-CPS-Association/into-cps-application/tree/development) with description of the development environment (versions of nodejs, npm, gulp etc.) and the gulp commands needed to execute the development flows and building an exe file. There is outdated documentation on [wiki](https://github.com/INTO-CPS-Association/into-cps-application/wiki/NPM-Gulp-Tasks). Let us consolidate and update this documentation into DEVELOPER.md. Thanks.

**Comment by prasadtalasila** [comment]
@PandoraQS I receive the following errors during packaging.

```sh
$gulp build

[10:55:46] Using gulpfile ~/Documents/git/into-cps/into-cps-application/gulpfile.js

[10:55:46] Starting 'build'...

[10:55:46] Starting 'compile-ts'...

[10:55:52] Finished 'compile-ts' after 6.04 s

[10:55:52] Starting 'compile-ng2'...

[10:55:54] Finished 'compile-ng2' after 1.75 s

[10:55:54] Starting 'copy-js'...

[10:55:54] Finished 'copy-js' after 39 ms

[10:55:54] Starting 'copy-html'...

[10:55:54] Finished 'copy-html' after 121 ms

[10:55:54] Starting 'copy-css'...

[10:55:54] Finished 'copy-css' after 462 ms

[10:55:54] Starting 'copy-fonts'...

[10:55:54] Finished 'copy-fonts' after 21 ms

[10:55:54] Starting 'copy-custom'...

[10:55:55] Finished 'copy-custom' after 43 ms

[10:55:55] Finished 'build' after 8.49 s

$gulp package
[10:56:32] Using gulpfile ~/Documents/git/into-cps/into-cps-application/gulpfile.js

[10:56:32] Starting 'package'...

[10:56:32] Starting 'clean'...

[10:56:33] Finished 'clean' after 50 ms

[10:56:33] Starting 'prep-pkg'...

[10:56:33] Starting 'compile-ts-uglify'...

[10:56:41] Finished 'compile-ts-uglify' after 8.44 s

[10:56:41] Starting 'compile-ng2'...

[10:56:43] Finished 'compile-ng2' after 2.35 s

[10:56:43] Starting 'copy-js'...

[10:56:43] Finished 'copy-js' after 121 ms

[10:56:43] Starting 'copy-html'...

[10:56:44] Finished 'copy-html' after 108 ms

[10:56:44] Starting 'copy-css'...

[10:56:44] Finished 'copy-css' after 496 ms

[10:56:44] Starting 'copy-fonts'...

[10:56:44] Finished 'copy-fonts' after 7.85 ms

[10:56:44] Starting 'copy-custom'...

[10:56:44] Finished 'copy-custom' after 54 ms

[10:56:44] Finished 'prep-pkg' after 12 s

[10:56:44] Starting 'pkg'...

Packaging app for platform linux x64 using electron v10.4.7

[10:57:03] Finished 'pkg' after 19 s

[10:57:03] Finished 'package' after 30 s

$./pkg/into-cps-app-4.0.8-dev-linux-x64/into-cps-app-4.0.8-dev

(electron) 'allowRendererProcessReuse' is deprecated and will be removed.

Npm start user data path: /home/dtaas/.config/INTO-CPS APP

{

  'RT-Tester Installation Path': 'C:/opt/rt-tester',

  'RT-Tester MBT Installation Path': 'C:/opt/rtt-mbt',

  'RT-Tester RTTUI3 Executable Path': 'C:/Program Files (x86)/Verified/RTTUI3/bin/rttui3.exe',

  'Python Executable Path for RT-Tester': 'C:/Python27/python.exe',

  update_site: 'https://raw.githubusercontent.com/INTO-CPS-Association/INTO-CPS-Association.github.io/master/download/',

  dev_update_site: 'https://raw.githubusercontent.com/INTO-CPS-Association/INTO-CPS-Association.github.io/development/download/',

  example_site: 'https://raw.githubusercontent.com/INTO-CPS-Association/INTO-CPS-Association.github.io/master/examples/examples.json',

  dev_example_site: 'https://raw.githubusercontent.com/INTO-CPS-Association/INTO-CPS-Association.github.io/development/examples/examples.json',

  development_mode: false,

  coe_host_url: 'localhost:8082',

  traceability_daemon_port: '8083',

  coe_remote_host: false,

  enable_traceability: false,

  local_update_site: '',

  use_local_update_site: false,

  graph_max_data_points: 1000,

  install_tmp_dir: '/home/dtaas/into-cps-projects/install_downloads',

  install_dir: '/home/dtaas/into-cps-projects/install',

  default_projects_folder_path: '/home/dtaas/into-cps-projects',

  active_project: '/home/dtaas/Documents/git/into-cps/into-cps-application/test/TestHelpers/../TestData/test9_data/project/.project.json'

}

Finished loading settings.

Running in development mode: false

Loading project from: /home/dtaas/Documents/git/into-cps/into-cps-application/test/TestHelpers/../TestData/test9_data/project/.project.json

Stored settings in : /home/dtaas/.config/INTO-CPS APP/intoCpsApp/settings.json.

Stored settings in : /home/dtaas/Documents/git/into-cps/into-cps-application/test/TestHelpers/../TestData/test9_data/project/.project.json.

[13841:0711/105828.120439:FATAL:gpu_data_manager_impl_private.cc(442)] GPU process isn't usable. Goodbye.

[1]    13841 trace trap (core dumped)  ./pkg/into-cps-app-4.0.8-dev-linux-x64/into-cps-app-4.0.8-dev
```

**Comment by prasadtalasila** [comment]
@PandoraQS Even `npm start` fails

```sh
$npm start

> into-cps-app@4.0.8-dev start /home/user/Documents/git/into-cps/into-cps-application

> electron dist/main.js

(electron) 'allowRendererProcessReuse' is deprecated and will be removed.

Npm start user data path: /home/dtaas/.config/Electron

Running in development mode: false

Stored settings in : /home/user/.config/Electron/intoCpsApp/settings.json.

[11573:0711/100242.982128:FATAL:gpu_data_manager_impl_private.cc(442)] GPU process isn't usable. Goodbye.

/home/user/Documents/git/into-cps/into-cps-application/node_modules/electron/dist/electron exited with signal SIGTRAP

npm ERR! code ELIFECYCLE

npm ERR! errno 1

npm ERR! into-cps-app@4.0.8-dev start: `electron dist/main.js`

npm ERR! Exit status 1

npm ERR! 

npm ERR! Failed at the into-cps-app@4.0.8-dev start script.

npm ERR! This is probably not a problem with npm. There is likely additional logging output above.



npm ERR! A complete log of this run can be found in:

npm ERR!     

> /home/user/.npm/_logs/2024-07-11T08_02_43_102Z-debug.log

```

**Pull request merged**